packages/opentelemetry-propagator-jaeger/README.md

```diff
@@ -0,0 +1,38 @@
+# OpenTelemetry Propagator Jaeger
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+OpenTelemetry propagation Jaeger provide HTTP header propagation for systems that using Jaeger HTTP header format.
```

Comment:
What would you say for such description?:
`OpenTelemetry Jaeger Propagator provides HTTP header propagation for systems that are using Jaeger HTTP header format.`